bench-vortex/src/bin/clickbench.rs

```diff
@@ -92,9 +92,9 @@ struct DuckDBCtx {
 
 impl DuckDBCtx {
     pub fn duckdb_file(&self, format: Format) -> PathBuf {
-        let dir = Path::new("bench-vortex/data/duckdb");
-        std::fs::create_dir_all(dir).vortex_expect("failed to create duckdb data dir");
-        dir.join(format!("{format}.db"))
+        let dir = format!("clickbench_partitioned/{}", format.name()).to_data_path();
```

Comment:
I think this should be relative to the Git root, not relative to current working dir.

Reply:
We changed this recently away from writing to temp.

Reply:
it is `to_data_path()`